tests/unit/replicator_tests.py

```diff
@@ -116,6 +118,7 @@ def test_replication_with_generated_id(self):
         clone = Replicator(self.client)
         clone.create_replication(self.db, self.target_db)
 
+    @flaky(max_runs=3)
```

Comment:
I went back into the Jenkins build history and only found one build on Oct 20th that had `test_create_replication` failing.  From where I'm standing, it seems like there's something else going on here.